tests/constbuffer_array_splitter_ut/constbuffer_array_splitter_ut.c

```diff
@@ -992,4 +1001,810 @@ TEST_FUNCTION(constbuffer_array_splitter_split_with_all_empty_buffers_fails_if_u
     real_constbuffer_array_dec_ref(buffers);
 }
 
+/* constbuffer_array_splitter_split_to_array_of_array */
+
+/* Tests_SRS_CONSTBUFFER_ARRAY_SPLITTER_07_001: [ If buffers is NULL then constbuffer_array_splitter_split_to_array_of_array shall fail and return NULL. ]*/
+TEST_FUNCTION(constbuffer_array_splitter_split_to_array_of_array_with_null_buffers_fails)
+{
+    /// act
```

Comment:
> /// act [](http://example.com/codeflow?start=3&length=8)

Missing /// arrage
We usually put it in the test for consistency, even if there are no actual arrange calls

Reply:
added